microsoft-edge/extensions-chromium/developer-guide/api-support.md

```diff
@@ -12,72 +12,72 @@ ms.date: 12/13/2023
+| [tabCapture](https://developer.chrome.com/docs/extensions/reference/api/tabCapture) | Interacts with tab media streams. | MV2, MV3 | Windows, Linux, Mac |
+| [tabGroups](https://developer.chrome.com/docs/extensions/reference/api/tabGroups) | Interacts with the browser's tab grouping system.  Modifies and rearranges tab groups in the browser.  To group and ungroup tabs, or to query what tabs are in groups, use the [tabs](https://developer.chrome.com/docs/extensions/reference/api/tabs) API instead. | MV3 | Windows, Linux, Mac |
+| [tabs](https://developer.chrome.com/docs/extensions/reference/api/tabs) | Interacts with the browser's tab system to create, modify, and rearrange tabs. | MV2, MV3 | Windows, Linux, Mac, Android |
+| [topSites](https://developer.chrome.com/docs/extensions/reference/api/topSites) | Accesses the top sites, also called the _most visited sites_, that are displayed on the new tab page. These sites don't include shortcuts that are customized by the user. | MV2, MV3 | Windows, Linux, Mac |
+| [tts](https://developer.chrome.com/docs/extensions/reference/api/tts) | Plays synthesized text-to-speech (TTS). | MV2, MV3 | Windows, Linux, Mac |
+| [ttsEngine](https://developer.chrome.com/docs/extensions/reference/api/ttsEngine) | Implements a text-to-speech (TTS) engine using an extension.  Extensions that register to use this API receive events that contain utterances to be spoken and other parameters.  Extensions can then use any available web technology to synthesize and output speech, and send events back to the calling function to report the status. | MV2, MV3 | Windows, Linux, Mac |
+| [types](https://developer.chrome.com/docs/extensions/reference/api/types) | Type declarations for Microsoft Edge. | MV2, MV3 | Windows, Linux, Mac |
+| [userScripts](https://developer.chrome.com/docs/extensions/reference/api/userScripts) | Executes user scripts in the User Scripts context. | MV3 | Windows, Linux, Mac, Android |
+| [webAuthenticationProxy](https://developer.chrome.com/docs/extensions/reference/api/webAuthenticationProxy) | Lets remote desktop software that's running on a remote host intercept Web Authentication API (WebAuthn) requests in order to handle them on a local client. | MV3 | Windows, Linux, Mac |
+| [webNavigation](https://developer.chrome.com/docs/extensions/reference/api/webNavigation) | Receives notifications about the status of navigation requests. | MV2, MV3 | Windows, Linux, Mac, Android |
+| [webRequest](https://developer.chrome.com/docs/extensions/reference/api/webRequest) | Observes and analyzes traffic.  Intercepts, blocks, or modifies requests. | MV2, MV3 | Windows, Linux, Mac, Android |
+| [windows](https://developer.chrome.com/docs/extensions/reference/api/windows) | Interacts with browser windows to create, modify, and rearrange windows in the browser. | MV2, MV3 | Windows, Linux, Mac |
 
 
```

Comment:
line 132 See Also section: link to an Android page?

Rendered PR branch > See Also:
https://github.com/BiliBoomBo/edge-developer/blob/patch-1/microsoft-edge/extensions-chromium/developer-guide/api-support.md#see-also --
The Chrome V3 linked page doesn't mention Android: 
[Chrome Extensions API reference for Manifest V3](https://developer.chrome.com/docs/extensions/reference/api/)

@BiliBoomBo @captainbrosset @bhuvanapriyap

Reply:
found in repo:

* [Android Studio] https://developer.android.com/sdk/installing/studio.html)
* [Android Accessibility Suite] https://play.google.com/store/apps/details?id=com.google.android.marvin.talkback
* [set up a virtual device] https://developer.android.com/tools/devices/managing-avds.html
* [start the emulator] https://developer.android.com/tools/devices/emulator.html
* [Android Accessibility Suite] https://play.google.com/store/apps/details?id=com.google.android.marvin.talkback
* ...

Reply:
@mikehoffms
Thanks for reviewing. This API list is based on our internal testing. Chrome on Android doesn't support extensions yet, but Edge on Android does. Therefore, there won’t be any mention of Android on the Chromium wiki. However, since extensions on Android are similar to those on desktop, we can refer to the desktop extension wiki for reference.